resource/get_language_list.py

```diff
@@ -12,5 +12,6 @@ def do(payload, config, plugin_config, inputs):
         language_choices = sorted(
             [{"value": k, "label": v} for k, v in SUPPORTED_LANGUAGES_PYCLD3.items()], key=lambda x: x.get("label")
         )
-        language_choices.insert(0, {"label": "None", "value": "None"})
+        if payload["parameterName"] == "fallback_language":
```

Comment:
What does it do ?

Reply:
If the parameter is "fallback_language" then it serves the `choices` dict